singlestoredb/fusion/handlers/stage.py

```diff
@@ -44,10 +46,10 @@ class ShowStageFilesHandler(SQLHandler):
 
     Arguments
     ---------
-    * ``<group_id>``: The ID of the workspace group in which
```

Comment:
You should use `-` in the names instead of `_` like quoted strings in the grammar. The value in the quoted string is what shows up in the Fusion help system.